test/general/test_containers.cpp

```diff
@@ -205,3 +205,9 @@ TEST(AnyMap, loadYaml)
     EXPECT_EQ(coeffs[0].size(), (size_t) 7);
     EXPECT_DOUBLE_EQ(coeffs[1][2], -8.280690600E-07);
 }
+
+TEST(AnyMap, loadDeprecatedYaml)
+{
+    make_deprecation_warnings_fatal();
```

Comment:
This is called automatically at the start of the test suite (see `main` as defined in `test/general/string_processing.cpp`).